### ISSUE TITLE
typstfmt doesn't seem to work on my side

### ISSUE THREAD

**Issue by Myriad-Dreamin** [Myriad-Dreamin/tinymist#28]
This is because I have not made that feature.

**Comment by KagaJiankui** [comment]
A temporary workaround is use VSCode Tasks to invoke `typstfmt` in command line.

```json
{
  "label": "Typstfmt",
  "type":"shell",
  "command": "typstfmt",
  "args": [
    "${fileBasenameNoExtension}.typ"
  ],
  "options": {
    "cwd": "${workspaceFolder}/${relativeFileDirname}"
  },
  "problemMatcher": []
}
```

**Comment by Myriad-Dreamin** [comment]
fixed by #113
